hexrd/ui/image_tab_widget.py

```diff
@@ -171,6 +171,14 @@ def update_ims_toolbar(self):
             self.toolbars[idx]['sb'].update_name(self.image_names[idx])
             self.toolbars[idx]['sb'].update_range(True)
 
+    def toggle_off_toolbar(self):
+        toolbars = [bars['tb'] for bars in self.toolbars]
+        for tb in toolbars:
+            if tb.mode == 'zoom rect':
+                tb.zoom()
+            if tb.mode == 'pan/zoom':
```

Comment:
Lets move these to the constants module, with suitable comment, as we are reaching into matplotlib internals!

Reply:
Sounds good, updated. Let me know if the comment seems sufficient